apps/client/app/purchase-orders/page.tsx

```diff
@@ -1,8 +1,112 @@
+      <table className="table">
+        <thead className="bg-slate-200">
+          <tr>
+            <th className="border-b border-slate-100 dark:border-slate-700 p-4 pl-8 text-slate-500 dark:text-slate-400 text-left">
+              Purchase Order Id
+            </th>
+            <th className="border-b border-slate-100 dark:border-slate-700 p-4 pl-8 text-slate-500 dark:text-slate-400 text-left">
+              Vendor
+            </th>
+            <th className="border-b border-slate-100 dark:border-slate-700 p-4 pl-8 text-slate-500 dark:text-slate-400 text-left">
+              Order Date
+            </th>
+            <th className="border-b border-slate-100 dark:border-slate-700 p-4 pl-8 text-slate-500 dark:text-slate-400 text-left">
+              Expected Delivery Date
+            </th>
+            <th className="border-b border-slate-100 dark:border-slate-700 p-4 pl-8 text-slate-500 dark:text-slate-400 text-left">
+              Total Item Quantity
+            </th>
+            <th className="border-b border-slate-100 dark:border-slate-700 p-4 pl-8 text-slate-500 dark:text-slate-400 text-left">
+              Total Price
+            </th>
+          </tr>
+        </thead>
+        <tbody className="bg-white dark:bg-slate-800">
+          {sortedPurchaseOrders.map((purchaseOrder: PurchaseOrder) => (
+            <tr key={purchaseOrder.id}>
+              <td className="border-b border-slate-100 dark:border-slate-700 p-4 pl-8 text-slate-500 dark:text-slate-400">
+                <Link
+                  href={'/purchase-orders/' + purchaseOrder.id}
+                  className="underline text-blue-600 hover:text-blue-800 visited:text-purple-600"
+                >
+                  {purchaseOrder.id}
+                </Link>
+              </td>
+              <td className="border-b border-slate-100 dark:border-slate-700 p-4 pl-8 text-slate-500 dark:text-slate-400">
+                {purchaseOrder.vendor_name}
+              </td>
+              <td className="border-b border-slate-100 dark:border-slate-700 p-4 pl-8 text-slate-500 dark:text-slate-400">
+                {moment(purchaseOrder.order_date).format('YYYY-MM-DD')}
+              </td>
+              <td className="border-b border-slate-100 dark:border-slate-700 p-4 pl-8 text-slate-500 dark:text-slate-400">
+                {moment(purchaseOrder.expected_delivery_date).format(
+                  'YYYY-MM-DD'
+                )}
+              </td>
+              <td className="border-b border-slate-100 dark:border-slate-700 p-4 pl-8 text-slate-500 dark:text-slate-400">
+                {purchaseOrder.purchase_order_line_items
+                  .map((item) => item.quantity)
+                  .reduce((prev, next) => prev + next)}
+              </td>
+              <td className="border-b border-slate-100 dark:border-slate-700 p-4 pl-8 text-slate-500 dark:text-slate-400">
+                <Currency
+                  value={purchaseOrder.purchase_order_line_items
```

Comment:
Feels like there's a small amount of business logic leaking into the display with this... What alternatives do we have to reduce the amount of data we are processing? How do we know if these totals are accurate?